scripts/lib/CIME/case/case_run.py

```diff
@@ -50,14 +50,14 @@ def _pre_run_check(case, lid, skip_pnl=False, da_cycle=0):
     # This needs to be done everytime the LID changes in order for log files to be set up correctly
     # The following also needs to be called in case a user changes a user_nl_xxx file OR an env_run.xml
     # variable while the job is in the queue
-    logger.debug("{} NAMELIST CREATION BEGINS HERE".format(time.strftime("%Y-%m-%d %H:%M:%S")))
+    model_log("e3sm", logger, "{} NAMELIST CREATION BEGINS HERE".format(time.strftime("%Y-%m-%d %H:%M:%S")))
```

Comment:
I don't think this should be hardcoded to e3sm.

Reply:
If I recall, you guys (cesm) did not want this output as logger.info, so I made it logger.debug. That messed-up folks on E3SM since they had come to expect it, so this was the best compromise.

Reply:
RIght but see my note below - now we don't get it even with logger set to debug

Reply:
Done.